python/tskit/tables.py

```diff
@@ -2873,10 +2881,10 @@ def __getstate__(self):
         return self.asdict()
 
     @classmethod
-    def load(cls, file_or_path):
+    def load(cls, file_or_path, skip_tables=False):
```

Comment:
```suggestion
    def load(cls, file_or_path, *, skip_tables=False):
```
Safer make this a keyword only arg